src/views/virtualmachines/list/utils/totalsQueries.ts

```diff
@@ -0,0 +1,27 @@
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+
+export const VMTotalsQueries = {
+  TOTAL_CPU_REQUESTED: 'TOTAL_CPU_REQUESTED',
+  TOTAL_CPU_USAGE: 'TOTAL_CPU_USAGE',
+  TOTAL_MEMORY_USAGE: 'TOTAL_MEMORY_USAGE',
+  TOTAL_STORAGE_CAPACITY: 'TOTAL_STORAGE_CAPACITY',
+  TOTAL_STORAGE_USAGE: 'TOTAL_STORAGE_USAGE',
+};
+
+export const getVMTotalsQueries = (namespace: string, namespacesList: string[]) => {
+  const isAllNamespaces = namespace === ALL_NAMESPACES_SESSION_KEY;
+
+  const byNamespace = isAllNamespaces ? '' : ' by (namespace)';
+  const namespaceFilter = isAllNamespaces ? '' : `{namespace='${namespace}'}`;
+  const namespaceFilterCpu = isAllNamespaces
```

Comment:
please adjust the names ot be more clear,
for example
orderByNamespace or aggragteByNamespace..